packages/cli/generators/relation/utils.generator.js

```diff
@@ -0,0 +1,228 @@
+ * @param {propertyName} string
+ * @param {propertyType} string
+ *
+ * @return bool true on success, false on failure.
+ */
+
+exports.isValidPropertyType = function(classObj, propertyName, propertyType) {
+  return this.getPropertyType(classObj, propertyName) == propertyType;
+};
+
+exports.doesParameterExist = function(classConstructor, parameterName) {
+  return classConstructor
+    .getParameters()
+    .map(x => x.getName())
+    .includes(parameterName);
+};
+
+exports.addForeignKey = function(foreignKey, sourceModelPrimaryKeyType) {
+  return {
+    decorators: [
+      {
+        name: 'property',
+        arguments: ["{\n type : '" + sourceModelPrimaryKeyType + "',\n}"],
+      },
+    ],
+    name: foreignKey + '?',
+    type: sourceModelPrimaryKeyType,
+  };
+};
+
+exports.addProperty = function(classOBj, property) {
+  classOBj.insertProperty(this.getPropertiesCount(classOBj), property);
+  classOBj.insertText(this.getPropertyStartPos(classOBj), '\n');
+};
+
+exports.getPropertiesCount = function(classObj) {
+  return classObj.getProperties().length;
+};
+
+exports.getPropertyStartPos = function(classObj) {
+  return classObj
+    .getChildSyntaxList()
+    .getChildAtIndex(this.getPropertiesCount(classObj) - 1)
+    .getPos();
+};
+
+exports.addRequiredImports = function(sourceFile, imports) {
+  for (let currentImport of imports) {
+    this.addCurrentImport(sourceFile, currentImport);
+  }
+};
+
+exports.getRequiredImports = function(targetModel, relationType) {
+  return [
+    {
+      name: targetModel,
+      module: './' + utils.kebabCase(targetModel) + '.model',
+    },
+    {
+      name: relationType,
+      module: '@loopback/repository',
+    },
+  ];
+};
+
+exports.addCurrentImport = function(sourceFile, currentImport) {
+  if (!this.doesModuleExists(sourceFile, currentImport.module)) {
+    sourceFile.addImportDeclaration({
+      moduleSpecifier: currentImport.module,
+    });
+  }
+  if (!this.doesImportExistInModule(sourceFile, currentImport)) {
+    sourceFile
+      .getImportDeclarationOrThrow(currentImport.module)
+      .addNamedImport(currentImport.name);
+  }
+};
+
+exports.doesModuleExists = function(sourceFile, moduleName) {
+  return sourceFile.getImportDeclaration(moduleName);
+};
+
+exports.doesImportExistInModule = function(sourceFile, currentImport) {
+  let identicalImport;
+  const relevantImports = this.getNamedImportsFromModule(
+    sourceFile,
+    currentImport.module,
+  );
+  if (relevantImports.length > 0) {
+    identicalImport = relevantImports[0]
+      .getNamedImports()
+      .filter(imp => imp.getName() == currentImport.name);
+  }
+
+  return identicalImport && identicalImport.length > 0;
+};
+
+exports.getNamedImportsFromModule = function(sourceFile, moduleName) {
+  const allImports = sourceFile.getImportDeclarations();
+  return allImports.filter(imp => imp.getModuleSpecifierValue() == moduleName);
+};
+
+exports.runPrettier = function(generatorObj, filename) {
```

Comment:
Please remove this helper and all places calling it, as discussed in https://github.com/strongloop/loopback-next/pull/2426/files#r266835677

Reply:
Done. Added --format option into relation cli to run lint:fix.